.travis.yml

```diff
@@ -36,10 +36,19 @@ script:
  - STACK_OPTS="$STACK_OPTS --extra-include-dirs=/usr/local/include --extra-lib-dirs=/usr/local/lib"
  - |
    set -ex
-   # Run tests
+   # Run tests using solc 0.4.25
    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib"
+   cp ~/.local/bin/solc-0.4.25 ~/.local/bin/solc || true
    stack --no-terminal test --ghc-options="$GHC_OPTIONS" $STACK_OPTS
    set +ex
+ - |
+   set -ex
```

Comment:
Rather than duplicating set +ex and export, let's just make a function that takes a binary, symbolic links it into $PATH as solc, and then runs stack tests. Then we can just run that function twice

Reply:
Another option is to implement a new config option: `solcBin` to select `solc-0.4.25` and `solc-0.5.4` during the compilation of a contract.

Reply:
Done (but untested, let's wait for travis..)

Reply:
Works as expected.